docs/source/changelog.rst

```diff
@@ -1,6 +1,33 @@
 What's new
 ==========
 
+einsteinpy 0.2.1 - 2019-09-15
+-----------------------------
+
+This minor release would bring improvements and new feature additions to the already existing symbolic calculations module.
+Part of this release is sponsored by European Space Agency, through Summer of Code in Space
```

Comment:
Can you add a newline between 1st and 2nd line?

Reply:
Sure :)